main.ml

```diff
@@ -511,9 +504,10 @@ let process root_dir name =
         | Some "xeno+shm" ->
           let uid = Uri.path uri in
           let uid = if String.length uid > 1 then String.sub uid ~pos:1 ~len:(String.length uid - 1) else uid in
-          RRD.Client.Plugin.Local.register ~uid ~info:Rrd.Five_Seconds ~protocol:Rrd_interface.V2
-          >>= fun _ ->
-          loop (uid :: acc) datasources
+          (RRD.Client.Plugin.Local.register RRD.rpc uid Rrd.Five_Seconds Rrd_interface.V2).Rpc_async.M.async
```

Comment:
The additional `RRD.rpc` is part of the client generated interface

Reply:
So it should be namespaced as `RRD.Client.rpc`?

Reply:
No, no need, it was just for me to understand :p